examples/polyglot-zipkin/README.adoc

```diff
@@ -11,22 +11,26 @@ of the endpoints might not work as they are calling external services. Feel free
 services.
 
 == Run
+Before running ensure that containers from previous run were removed.
+
 [source,shell]
 ----
+$ export KAFKA_ZOOKEEPER=172.17.0.1
 $ export TRACING_SERVER=172.17.0.1
 $ export TRACING_PORT=8080
+
 $ docker-compose pull
 $ docker-compose up
-$ docker-compose down -> stop and remove
+$ docker-compose down -> stop and remove all containers
 
--> Run tracing server (Hawkular APM or zipkin server). Server has to be started after containers to make sure kafka is running.
--> If using Hawkular APM use /standalone.sh -b 0.0.0.0.
--> Zipkin server has to be started with KAFKA_ZOOKEEPER=172.17.0.1:2181 java -jar zipkin-server.jar
```

Comment:
This was added with python example. I added it to explicitly say that it is required to set `KAFKA_ZOOKEEPER`, but I have moved it to upper section with `export` to tracing server can be executed in 'a normal way'.